### PR TITLE
More normalizers

### PR DESCRIPTION
**Pull request by orangejulius**
Adds normalizers to remove ordinals and begin abbreviating street suffexes

@stephenkhess please review :)

### CODE REVIEW

**Comment by orangejulius** [comment]
I added one more street abbreviation but there are certainly more. However these would at least get us started in that direction. @stephenkhess can you think of a few more to add?

File: lib/normalizers.js [hunk 1]
```diff
@@ -55,6 +55,30 @@ module.exports.abbreviateDirectionals = function(val) {
 
 };
 
+module.exports.abbreviateStreetSuffexes = function(val) {
```
Comment (trescube) [inline lib/normalizers.js]: "suffexes" is not a word :)

File: lib/normalizers.js [hunk 2]
```diff
@@ -55,6 +55,30 @@ module.exports.abbreviateDirectionals = function(val) {
...
+    return val;
+  }
+
+  var terms = {
+    'STREET': 'ST',
+    'AVENUE': 'AVE',
+    'BOULEVARD': 'BLVD',
+    'ROAD': 'RD',
+    'DRIVE': 'DR'
+  };
```
Comment (trescube) [inline lib/normalizers.js:69]: See https://github.com/pelias/schema/blob/master/street_suffix.js for a complete list of street types/abbreviations mined from Tiger data and my memory.

File: lib/normalizers.js (hunk 1 above)
Comment (orangejulius) [inline lib/normalizers.js]: lol, fixed!

**Pull request merged**